config/locales/en/dashboard.yml

```diff
@@ -16,6 +16,18 @@ en:
         in_progress: In progress
         submitted: Submitted
         returned: Returned
+      sub_navigation:
+        link_1: In progress %{count}
+        link_2: Submitted %{count}
+        link_3: Returned %{count}
+    show:
```

Comment:
I think this `show` action is no longer in this controller but instead in the `completed_applications` one so we can remove these locales. Perhaps were reintroduced in a rebase, sorry about that!